modules/logging-release-notes-5-9-11.adoc

```diff
@@ -0,0 +1,37 @@
+// Module included in the following assemblies:
+//
+// * observability/logging/logging_release_notes/logging-5-9-release-notes.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="logging-release-notes-5-9-11_{context}"]
+= Logging 5.9.11
+
+This release includes link:https://access.redhat.com/errata/RHSA-2025:1227[RHSA-2025:1227].
+
+[id="openshift-logging-5-9-11-enhancements_{context}"]
+== Enhancements
+* This enhancement adds `OTel` semantic stream labels to the `lokiStack` output so that you can query logs by using both `ViaQ` and `OTel` stream labels. (link:https://issues.redhat.com/browse/LOG-6581[LOG-6581])
+
+[id="logging-release-notes-5-9-11-bug-fixes_{context}"]
+== Bug Fixes
+* Before this update, the collector container mounted all log sources. With this update, it mounts only the defined input sources. (link:https://issues.redhat.com/browse/LOG-5691[LOG-5691])
+
+* Before this update, a regression from version 5.8 caused Fluentd to support HTTP proxy, as originally reported in link:https://issues.redhat.com/browse/LOG-5466[LOG-5466]. With this update, you fix the regression, ensuring Fluentd no longer supports HTTP proxy.
```

Comment:
I don't think this conveys the correct message. The fix was not "to no longer support HTTP proxy", but to support the `no_proxy` settings. Keeping with our usual pattern, my suggestion would be:

```suggestion
* Before this update, fluentd ignored the `no_proxy` setting when using the HTTP output. With this update, the `no_proxy` setting is picked up correctly.
```